dozer-api/src/rest/mod.rs

```diff
@@ -165,18 +165,21 @@ impl ApiServer {
         shutdown: impl Future<Output = ()> + Send + 'static,
         labels: LabelsAndProgress,
     ) -> Result<Server, ApiInitError> {
+        let dozer_master_secret = std::env::var("DOZER_MASTER_SECRET").ok();
+        let security = match (dozer_master_secret, self.security) {
+            (Some(master_secret), None) => Some(ApiSecurity::Jwt(master_secret)),
+            (_, security) => security,
+        };
```

Comment:
This logic is different from what's used in `client_server.rs`. Here environment variable overrides configuration.

Reply:
hmm, i think the code above, logic will be: if api security is empty, we will take value from variable DOZER_MASTER_SECRET => which mean we prior config over variable

Reply:
Oh sorry my bad. Again we should create a function to avoid repeating this bit of logic